src/main/java/app/gpx_animator/TrackIcon.java

```diff
@@ -28,8 +28,8 @@
     private static final ResourceBundle RESOURCE_BUNDLE = Preferences.getResourceBundle();
 
     @NonNls
-    private static final String[] KEYS = {"airplane", "bicycle", "bus", "car", "jogging", "riding", "sailing", "ship", "train", "tramway",
-            "trekking"};
+    private static final String[] KEYS = {"airplane", "bicycle", "bus", "car", "jogging", "motorcycleleft", "motorcycleright",
+           "riding", "sailing", "ship", "train", "tramway", "trekking"};
```

Comment:
Please add only one generic motorcycle. I created issue #228 as a generic approach to turn the icons automatically in the driving direction.